CHANGELOG.md

```diff
@@ -6,8 +6,20 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+<!-- Update date and version value -->
+
```

Comment:
Should remove this comment since it's already updated at line 13 below for the release.